### DISCUSSION TITLE
List of Log Entries and types

### DISCUSSION THREAD

**Discussion by tiredofit** [stalwartlabs/smtp-server#38] [Q&A]
I am looking to move away from logging to file and would like to take advantage of centralized logging with Loki or Kibana. In the past I have used regular expressions to match patterns and expressions. I was wondering if the logging output could be detailed at some point so that I could build such routines.

**Comment by mdecimus** [comment]
If Loki or Kibana support OpenTelemetry, then you should be able to accomplish this.

**Reply by tiredofit** [reply]
Thanks! I will look into OpenTelemetry.
